### PR TITLE
Adding Prompt Engineering Hindi course to the courses section

### PR DESCRIPTION
**Pull request by Aashu-11**
Dear Repository Owner,

I hope you're well. I'm a beginner participating in Hacktoberfest 2024 and came across your project. I’ve added a link to the Hindi version of the Prompt Engineering course to make it more accessible for Hindi-speaking users.

I’d appreciate it if you could review and consider merging my changes.

Thank you for your time and support!

Best regards,
Aayush Kolte.

**What does this PR do?**

This PR adds the Prompt engineering course in hindi for beginners under artifical intelligence subsection of courses section.

 **Why is this valuable ?**

This prompt engineering course is valuable because it teaches how to effectively communicate with AI models whose use is growing drastically everyday, optimizing outputs for specific tasks. It enhances problem-solving by crafting precise, clear, and efficient prompts. Mastering this skill improves productivity and accuracy in AI-driven solutions. Additionally, it fosters a deeper understanding of model behavior, helping to leverage AI more effectively.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Aashu-11** [comment]
Please approve my request

**Comment by eshellman** [comment]
Don't invent titles! Use the title given by the resource.

**Comment by Aashu-11** [comment]
@eshellman I have updated course name as per resource...so please do consider and merge my request as soon as possible

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -141,6 +141,7 @@
 * [Artificial Intelligence (Complete Playlist)](https://www.youtube.com/playlist?list=PLxCzCOWd7aiHGhOHV-nwb0HR5US5GFKFI) - Gate Smashers
 * [Generative AI Series - Showcasing Generative AI By Building Projects](https://www.youtube.com/playlist?list=PLu0W_9lII9aiS4rUVp2jXwIvCruo27sG6) - CodeWithHarry
 * [Playlist to Artificial Intelligence ](https://www.youtube.com/playlist?list=PLPIwNooIb9vgB1DQEftkKA3qOdeC4vonA) - Perfect Computer Engineer
+* [Prompt Engineering Full Course in Hindi (beginner to Master)] (https://www.youtube.com/playlist?list=PLyz4Eb45WBQ02Md7BiIO1sUsKTs8GcWKS) - EduExpress
```
Comment (eshellman) [inline courses/free-courses-hi.md:144]:
```suggestion
* [Prompt Engineering Full Course in Hindi (beginner to Master)](https://www.youtube.com/playlist?list=PLyz4Eb45WBQ02Md7BiIO1sUsKTs8GcWKS) - EduExpress
```

**Comment by Aashu-11** [comment]
@eshellman I had committed the changes as said by you again after removing spaces pls approve and merge my request

**Comment by Aashu-11** [comment]
@eshellman please merge my request I had made necessary changes. Please merge it as soon as possible.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**